src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.scss

```diff
@@ -240,6 +240,7 @@
     .datatable-row-detail {
       padding: 20px;
       border-bottom: 2px solid $color-table-header-border;
+      overflow-y: auto !important;
```

Comment:
Using `visible `fixes the problem.
```suggestion
      overflow-y: visible !important;
```

Reply:
It was my initial approach , but in case of just 1 snapshot in the list , drop-down view is still hidden
![Screenshot from 2020-06-07 18-50-18](https://user-images.githubusercontent.com/23611106/83969870-0c49c500-a8f0-11ea-8faf-d53141c53f2a.png)

Reply:
I have the opposite, with `auto` I get a scroll bar in the details and with `visible` I can see everything.

with `overflow-y: auto !important;`
![image](https://user-images.githubusercontent.com/399326/84037526-3b167880-a98e-11ea-9d6e-d64f9c95d493.png)

with `overflow-y: visible !important;`
![image](https://user-images.githubusercontent.com/399326/84037462-25a14e80-a98e-11ea-993a-2e62130c47aa.png)

Reply:
Yes, I agree with that . But the problem is when you create a snapshot and then see the drop-down, the view is hidden. So the problem occurs when we have just 1 snapshot in the list. I shared the screenshot regarding the same (in that I have 1 snapshot created)

Reply:
I was able to reproduce that, not sure why the size of the dropdown increases when you select a snapshot...
Guess the only solution is to use `container=body` and change the unit test...